bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1928,6 +1928,12 @@ Filename of input stream data for LAI
 Filename of input stream data for LAI
 </entry>
 
+<entry id="lai_dtlimit" type="real" category="datasets"
+       group="lai_streams" valid_values="" >
+dtlimit for LAI streams 
```

Comment:
Suggest adding a definition of dtlimit (ratio of max/min stream delta times). Also also add a bit for why you might increase it, which here is to allow cycling over a year of data. When it cycles back to the beginning it thinks the dt ratio is higher than it really is.

Reply:
done